tools/cldr-code/src/main/java/org/unicode/cldr/util/CollatorHelper.java

```diff
@@ -0,0 +1,32 @@
+package org.unicode.cldr.util;
+
+import com.ibm.icu.text.Collator;
+import com.ibm.icu.text.RuleBasedCollator;
+import com.ibm.icu.util.ULocale;
+
+public final class CollatorHelper {
+    public static final Collator ROOT_ORDER = makeRootOrder();
```

Comment:
```suggestion
    public static final RuleBasedCollator ROOT_ORDER = makeRootOrder();
```
We can make all of these RBCs, to avoid casting at call sites.